sql/_13_issues/_10_1h/cases/bug_2894.sql

```diff
@@ -1,83 +1,83 @@
 -- TRUE
-prepare s from 'select 1 from db_root where ?<>0' 
-execute s using true
+prepare s from 'select 1 from db_root where ?<>0';
+execute s using true;
 deallocate prepare s;
 
 -- FALSE
-prepare s from 'select 1 from db_root where ?<>0' 
-execute s using false
+prepare s from 'select 1 from db_root where ?<>0';
+execute s using false;
 deallocate prepare s;
 
 -- AND
-prepare s from 'select 1 from db_root where ?<>0 and ?<>0' 
-execute s using true, true
+prepare s from 'select 1 from db_root where ?<>0 and ?<>0';
+execute s using true, true;
 deallocate prepare s;
 
-prepare s from 'select 1 from db_root where ?<>0 and ?<>0' 
-execute s using true, false
+prepare s from 'select 1 from db_root where ?<>0 and ?<>0';
+execute s using true, false;
 deallocate prepare s;
 
-prepare s from 'select 1 from db_root where ?<>0 and ?<>0' 
+prepare s from 'select 1 from db_root where ?<>0 and ?<>0';
 execute s using false, true
```

Comment:
You need a semicolon at the end of the line.
`execute s using false, true;`

Reply:
This review has been reflected.